README.md

```diff
@@ -1,10 +1,13 @@
-![Licence](https://img.shields.io/badge/licence-AGPL--3-blue.svg)
 [![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/92/13.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-account-financial-tools-92)
-[![Build Status](https://travis-ci.org/OCA/account-financial-tools.svg?branch=13.0)](https://travis-ci.org/OCA/account-financial-tools)
-[![Coverage Status](https://coveralls.io/repos/OCA/account-financial-tools/badge.svg?branch=13.0)](https://coveralls.io/r/OCA/account-financial-tools?branch=13.0)
+[![Build Status](https://travis-ci.com/OCA/account-financial-tools.svg?branch=13.0)](https://travis-ci.com/OCA/account-financial-tools)
```

Comment:
If this is using GH actions, Travis shouldn't be here

Reply:
Ok, that's a bug in the template. I'll open a PR, but in the meantime we can merge this to unblock CI's...